src/irc.py

```diff
@@ -90,17 +89,18 @@ def prepare(self) -> None:
             snap_name=IRC_BRIDGE_SNAP_NAME,
             snap_channel=SNAP_PACKAGES[IRC_BRIDGE_SNAP_NAME]["channel"],
         )
-
         if not IRC_BRIDGE_CONFIG_DIR_PATH.exists():
             IRC_BRIDGE_CONFIG_DIR_PATH.mkdir(parents=True)
             logger.info("Created directory %s", IRC_BRIDGE_CONFIG_DIR_PATH)
             shutil.copy(IRC_BRIDGE_TEMPLATE_CONFIG_FILE_PATH, IRC_BRIDGE_CONFIG_DIR_PATH)
 
-        if not IRC_BRIDGE_UNIT_FILE_PATH.exists() or not IRC_BRIDGE_TARGET_FILE_PATH.exists():
-            shutil.copy(IRC_BRIDGE_TEMPLATE_UNIT_FILE_PATH, SYSTEMD_DIR_PATH)
-            shutil.copy(IRC_BRIDGE_TEMPLATE_TARGET_FILE_PATH, SYSTEMD_DIR_PATH)
-            systemd.daemon_reload()
-        systemd.service_enable(IRC_BRIDGE_SNAP_NAME)
+        self._generate_media_proxy_key()
+
+        with open(ENVIRONMENT_OS_FILE, "r+", encoding="utf-8") as env_file:
+            lines = env_file.read()
+            if "SNAP_MATRIX_APPSERVICE_ARGS" not in lines:
+                env_file.seek(0, os.SEEK_END)
```

Comment:
This is not necessary I believe. The previous `read()` call will already position the cursor at the end of the file.

Reply:
Good point. I removed and added a '\n' just in case in front of "SNAP_MATRIX_APPSERVICE_ARGS"